backend/core/test/applications/applications.e2e-spec.ts

```diff
@@ -10,6 +10,7 @@ import { UserModule } from "../../src/user/user.module"
 import { ListingsModule } from "../../src/listings/listings.module"
 import { EntityNotFoundExceptionFilter } from "../../src/filters/entity-not-found-exception.filter"
```

Comment:
I think the linter isn't running on this file - these imports aren't used now, they should be removed (and the linter should catch that...)